frontend/src/container/TopNav/index.tsx

```diff
@@ -8,38 +8,36 @@ import ShowBreadcrumbs from './Breadcrumbs';
 import DateTimeSelector from './DateTimeSelection';
 import { Container } from './styles';
 
-const routesToSkip = [
-	ROUTES.SETTINGS,
-	ROUTES.LIST_ALL_ALERT,
-	ROUTES.TRACE_DETAIL,
-	ROUTES.ALL_CHANNELS,
-	ROUTES.USAGE_EXPLORER,
-	ROUTES.INSTRUMENTATION,
-	ROUTES.VERSION,
-	ROUTES.ALL_DASHBOARD,
-	ROUTES.ORG_SETTINGS,
-	ROUTES.ERROR_DETAIL,
-	ROUTES.ALERTS_NEW,
-	ROUTES.EDIT_ALERTS,
-	ROUTES.LIST_ALL_ALERT,
+const skipDateTimeSelectorRoutes = [
+	'/settings',
```

Comment:
@hussamkhatib can you pick route from ROUTES object ?

Reply:
yes.
I thought so, But the eg mentioned in issue was done using absolute values of it.


> need to refactor the logic to display according to the route where this component is required
I din't quiet get what the that means

Reply:
Also, rolled back to use the Routes object